### PR TITLE
Release widget rc

### LINKED ISSUE #3972: [widget] Release widget-lib and widget-react

### PR DESCRIPTION
**Pull request by anxolin**
# Summary
Closes #3972 

This PR:
- [x] Makes events lib publishable (and publishes a new version)
- [x] Makes types lib publishable (and publishes a new version)
- [x] Publishes RCs for the libraries, so I can do a test in a App I created to test the widget for Safe end to end: https://github.com/anxolin/safe-wallet-web

## Context
I created this PR for experimenting with using the library from in a external Safe App

It is not the same to consume the NX package from our configurator app, since NX handles the magic. 

By testing to use it externally, I can see that some types are not exported, this is because they come from the events, and types packages which were not publishable. 

This PR makes them publishable. 

I will keep it in DRAFT until we have a release we are happy to close. For now publishing Release Candidates.

https://github.com/anxolin/safe-wallet-web

## edit 2: 01/03/2024 - Test the Toast Messages
After some  struggles, here it is a toast event on the Safe App, handled natively 🎉

Screenshot at Mar 01 21-26-50

## Edit 3: 05/03/2024 - Test the `partnerFee` and using `baseUrl`

I rebased to the current state of the the widget, and thanks to https://github.com/cowprotocol/cowswap/pull/3991 I was able to test the full end-to-end for the widget from a Safe app. 

I just pointed the widget to the PR branch, so I can see the the new fee.

Now I can test that with no fee, shows `FREE`

image

But if I add it:
image

## CONCLUSION
I would love to close the version for the libraries and leave it here for this widget version.

### CODE REVIEW

File: .github/workflows/npm.yml
Comment (anxolin) [inline .github/workflows/npm.yml:67]: @alfetopito , I know I added this somewhere. This made it work

File: libs/events/package.json
Comment (anxolin) [inline libs/events/package.json:3]: Once I get approvals, I will close versions for the 4 libs published here: types, events, and the 2 widget libs

**Review by shoom3301 — APPROVED** [review]

File: .github/workflows/npm.yml
Comment (alfetopito) [reply .github/workflows/npm.yml:67]: Hum, it defaults to `restricted`

> [--access <public|restricted>] Tells the registry whether this package should be published as public or restricted. Only applies to scoped packages, which default to restricted. If you don't have a paid account, you must publish with --access public to publish scoped packages.

https://docs.npmjs.com/cli/v6/commands/npm-publish

File: libs/events/project.json
Comment (alfetopito) [inline libs/events/project.json:2]: Why remove the namespace?

**Review by alfetopito — APPROVED** [review]

File: libs/events/project.json
Comment (anxolin) [reply libs/events/project.json:2]: Cause that's the project name within NX. you don't want to use the full qualified name when building or executing any task. Also, other NX projects don't have it

**Pull request merged**
